carma-messenger-config/example_calibration_folder/vehicle/calibration/urdf/carma.urdf

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0"?>

```

Comment:
Can we update this to match the carma.urdf used by the Chevrolet Tahoe for CARMA Messenger, since CARMA Messenger really only needs the urdf to describe the location of the Torc Pinpoint currently?

https://github.com/usdot-fhwa-stol/carma-vehicle-calibration/blob/develop/2018-chevrolet-tahoe/vehicle/calibration/urdf/carma.urdf